### ISSUE TITLE
Add setting for game folder location (mods and saves)

### ISSUE THREAD

**Issue by OtakuDeith** [PavelZinchenko/event-horizon-main#487]

**Comment by JVDP-LNS** [comment]
I agree we need to have an option to select a publicly accessible folder for mod files, because the newer Android versions do not allow convenient access to the data folders of applications, leading to use of unconventional methods of file access which may not always work or may not be available for the user.

**Comment by PavelZinchenko** [comment]
Possible duplicate of #160, which requests configurable mod-folder location; #487 additionally mentions saves.
